flaml/nlp/huggingface/trainer.py

```diff
@@ -7,13 +7,49 @@
     TFTrainer = object
```

Comment:
```suggestion
    Seq2SeqTrainer = object
```
Remove import of Trainer